api/app/utils/fba_calculator.py

```diff
@@ -76,11 +77,20 @@ def __init__(self,  # pylint: disable=too-many-arguments
 def calculate_fire_behavour_advisory(station: FBACalculatorWeatherStation) -> FireBehaviourAdvisory:
     """ Transform from the raw daily json object returned by wf1, to our fba_calc.StationResponse object.
     """
-    # time of interest will be the same for all stations
+    # time of interest will be the same for all stations.
     time_of_interest = get_hour_20_from_date(station.time_of_interest)
 
-    fmc = cffdrs.foliar_moisture_content(station.lat, station.long, station.elevation,
-                                         get_julian_date(time_of_interest))
+    if station.fuel_type == 'C1':
+        # It seems REDapp uses a date of 144 for C1. The source of this value is unknown.
```

Comment:
were you able to determine that 144 value just by looking at the redapp code, or did you have to play around with inputs to deduce it?

Reply:
Play with inputs. I couldn't find the magic value anywhere. It could be that it only works for this test case - so really I should add a few more test cases for C1 as part of this PR. I'll do that on Tuesday.

`
FMC_Calc = jnius.autoclass('ca.cwfgm.fuel.FMC_Calc')
for i in range(0, 365):
    fmc_calc = FMC_Calc()
    fmc_calc.setAttributeValue(16801, i)
    result = fmc_calc.fmc(lat, long, elv, day_of_year)
    print(f'REDapp: {i} fmc:{result}')
`

Reply:
Updated the comment here after feedback from D.